src/libraries/System.Net.Http/tests/FunctionalTests/System.Net.Http.Functional.Tests.csproj

```diff
@@ -48,6 +50,9 @@
     <NodeNpmModule Include="node-fetch" />
     <NodeNpmModule Include="node-abort-controller" />
 
+    <RuntimeHostConfigurationOption Include="System.Net.Http.EnableActivityPropagation" Value="true" Trim="true" />
+    <RuntimeHostConfigurationOption Include="System.Net.Http.EnableMetrics" Value="true" Trim="true" />
```

Comment:
@pavelsavara do we have another case where this won't work during the dev loop? (at least for now)

Reply:
Indeed, same https://github.com/dotnet/runtime/issues/97449

Reply:
Only blazor dev loop. The non-blazor WASM should work I think.

Reply:
> Only blazor dev loop. The non-blazor WASM should work I think.

Only runtime tests are loading the runtime config.bin